test/library/packages/Socket.precomp

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+pkg-config --libs --cflags libevent > 'Socket.compopts'
```

Comment:
If we're generating the `.compopts` file, it should not be checked in (and should instead be listed in a .gitignore, though I can never remember if we want it to be a global or directory .gitignore)

Reply:
directory-local is preferred so that the directory can be moved around / removed / etc. without changing a global file.